graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -249,3 +249,44 @@ def test_graph_construct(gds: GraphDataScience) -> None:
     assert G.relationship_count() == 4
 
     G.drop()
+
+
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 1, 0))
+def test_graph_construct_multiple_dfs(gds: GraphDataScience) -> None:
+    nodes = [pandas.DataFrame({"node_id": [0, 1]}), pandas.DataFrame({"node_id": [2, 3]})]
+    relationships = pandas.DataFrame({"source_id": [0, 1, 2, 3], "destination_id": [1, 2, 3, 0]})
```

Comment:
`destination_id` has been renamed to `target_id`

Reply:
done